python/vulns_per_assets/page_assets.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import requests
+
+# Might have to change this
+base_url = "https://api.kennasecurity.com/assets"
+
+def get_asset_page(page_num):
+    page_param = "?page=" + str(page_num)
+    url = base_url + page_param
+    
+    # Obtain the specified page.
+    try:
+        response = requests.get(url, headers=headers)
+    except Exception as exp:
+        print("List Asset Error: " + exp.__str__())
+        sys.exit(1)
+    
+    if response.status_code != 200:
+        print(f"List Asset Error: {response.status_code}")
+        print(f"Messages: {response.json()['message']}")
+        sys.exit(1)
+    
+    return response.json()
+
+# main
```

Comment:
Consider wrapping in `if __name__ == "__main__"`

Reply:
Also modified.